src/FunnelChart/FunnelSeries/FunnelArc.tsx

```diff
@@ -59,6 +65,11 @@ export interface FunnelArcProps {
    */
   gradient: ReactElement<GradientProps, typeof Gradient> | null;
 
+  /**
+   * Glow element for the arc.
+   */
+  glow?: ReactElement<GlowProps, typeof Glow> | null;
```

Comment:
We are not using glow?

Reply:
I used it to build the `glowStyles` below. I just copied the way it's done in other components like BubbleChart for example https://github.com/reaviz/reaviz/blob/master/src/BubbleChart/Bubble.tsx#L104-L106

Reply:
Let's cleanup that bubble one too w/ whatever we decide here.